### ISSUE TITLE
Main navigation needs proper WCAG fixes

### ISSUE THREAD

**Issue by jjlankinen** [otavanopisto/muikku#5127]
Main navigation needs to have proper role, aria-* and tabindex attributes set to correct elements.

This includes profile and locale selection drop down menus as well.

Following page describes how they should be set: https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html

Child of #5079
